src/Stack/Types/PackageName.hs

```diff
@@ -10,20 +10,13 @@
 -- | Names for packages.
 
 module Stack.Types.PackageName
-  (PackageName
-  ,PackageNameParseFail(..)
-  ,parsePackageName
-  ,parsePackageNameThrowing
+  (parsePackageNameThrowing
```

Comment:
How about moving this function to `pantry`? See any problem with that?

Reply:
Actually I've done that and then returned it back, I guess you're right and it would be better to have parsing code in 1 place